### PR TITLE
Update external-library.md

### PR DESCRIPTION
**Pull request by FedericoCalzoni**
I believe that displaying the code for both sections, even if it seems a bit repetitive, can help prevent fast readers from overlooking it

### CODE REVIEW

File: docs/docs/guides/external-library.md [hunk 1]
```diff
@@ -6,15 +6,19 @@ in a directory on the same machine.
 
 # Mount the directory into the containers.
 
-Edit `docker-compose.yml` to add two new mount points under `volumes:`
+Edit `docker-compose.yml` to add two new mount points in the sections `immich-server:` and `immich-microservices:` under `volumes:`
 
-```
-  immich-server:
+```yaml
```
Comment (mmomjian) [inline docs/docs/guides/external-library.md:11]:
```suggestion
```diff
```

File: docs/docs/guides/external-library.md [hunk 2]
```diff
@@ -6,15 +6,19 @@ in a directory on the same machine.
...
+immich-server:
+    volumes:
+        - ${EXTERNAL_PATH}:/usr/src/app/external
```
Comment (mmomjian) [inline docs/docs/guides/external-library.md:14]:
```suggestion
+       - ${EXTERNAL_PATH}:/usr/src/app/external
```

File: docs/docs/guides/external-library.md [hunk 3]
```diff
@@ -6,15 +6,19 @@ in a directory on the same machine.
...
+
+immich-microservices:
     volumes:
-      - ${EXTERNAL_PATH}:/usr/src/app/external
+        - ${EXTERNAL_PATH}:/usr/src/app/external
```
Comment (mmomjian) [inline docs/docs/guides/external-library.md:18]:
```suggestion
+       - ${EXTERNAL_PATH}:/usr/src/app/external
```

File: docs/docs/guides/external-library.md [hunk 4]
```diff
@@ -6,15 +6,19 @@ in a directory on the same machine.
...
 ```
 
-Be sure to add exactly the same line to both `immich-server:` and `immich-microservices:`.
+Be sure to add exactly the same path.
```
Comment (mmomjian) [inline docs/docs/guides/external-library.md:21]:
```suggestion
Be sure to add exactly the same path to both services.
```

**Review by mmomjian — CHANGES_REQUESTED** [review]
thank you for this change

**Review by mmomjian — APPROVED** [review]

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**